packages/account/src/Components/poa/status/verified/verified.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import classNames from 'classnames';
-import { PropTypes } from 'prop-types';
 import { Icon } from '@deriv/components';
 import { PlatformContext } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import { PoiButton } from 'Components/poi/poi-button/poi-button.jsx';
 import IconMessageContent from 'Components/icon-message-content';
-import { ContinueTradingButton } from 'Components/poa/continue-trading-button/continue-trading-button.jsx';
+import { ContinueTradingButton } from 'Components/poa/continue-trading-button/continue-trading-button';
+import { TPlatformContext } from 'Types';
 
-export const Verified = ({ needs_poi, is_description_enabled = true }) => {
-    const { is_appstore } = React.useContext(PlatformContext);
+type TVerified = {
```

Comment:
we have the same type in the poa/status/needs-review/needs-review.tsx file you can export it from there